indra/assemblers/indranet/assembler.py

```diff
@@ -0,0 +1,147 @@
+import logging
+import pandas as pd
+from .net import IndraNet
+from indra.statements import *
+from itertools import permutations
+from collections import OrderedDict
+
+
+logger = logging.getLogger(__name__)
+NS_PRIORITY_LIST = ('FPLX', 'HGNC', 'GO', 'MESH', 'HMDB', 'CHEBI', 'PUBCHEM')
```

Comment:
We could follow the priority used in Agent.get_grounding here, which would be: FPLX, HGNC, UP, CHEBI, GO, MESH. Other groundings like HMDB and PUBCHEM can be in arbitrary order at the end of the list. In practice, this may make a difference in at least two cases: chemicals that have both MESH and CHEBI grounding, there we want to prioritize CHEBI, and UP-grounded proteins that don't have human gene grounding.

Reply:
@dianakolusheva fixed it in https://github.com/sorgerlab/indra/pull/913/commits/e9f64393873dc274725dc331976fb33664fcf0d9